### ISSUE TITLE
Airship.activeNotifications not parsing on iOS

### ISSUE THREAD

**Issue by tomalabaster** [urbanairship/airship-flutter#155]
# Preliminary Info

### What Airship dependencies are you using?

`airship_flutter: 6.2.1`

### What are the versions of any relevant development tools you are using?

N/A

# Report

### What unexpected behavior are you seeing?

On iOS, when making a call to `Airship.activeNotifications`, the Future completes with the correct number of items however all properties on each `Notification` object in the list are null.

In `Notification._internal` (`lib/src/airship_flutter.dart` line 79), the properties in the `json` parameter for one of our given notifications when received on iOS are:

```
"aps" -> Map (2 items)
"^d" -> "our deep link URL"
"_" -> "e9cbf939-..."
"com.urbanairship.interactive_actions" -> Map (2 items)
"com.urbanairship.metadata" -> "..."
```

On Android, they are:

```
"alert" -> "our alert"
"title" -> "our title"
"subtitle" -> "our summary"
"extras" -> Map (9 items)
"notification_id" -> "our id"
"notificationId" -> "our id"
```

### What is the expected behavior?

Android and iOS decode the notifications in the same way.

### What are the steps to reproduce the unexpected behavior?

1. Send a notification from the Airship dashboard to both platforms
2. Launch the app without tapping on the notification
3. When making a call to `Airship.activeNotifications`, note using an appropriate method the contents of the list of notifications returned

### Do you have logging for the issue?

N/A

**Comment by rlepinski** [comment]
Should be fixed in 6.2.2

**Comment by tomalabaster** [comment]
Yep seems great, thanks @rlepinski! 🙌
